apps/hash-integration-worker/src/linear-activities.ts

```diff
@@ -116,51 +137,38 @@ const createHashEntity = async (params: {
     },
   });
 
-  const linkEntities = await HashEntity.createMultiple(
-    graphApiClient,
-    { actorId: params.authentication.actorId },
-    params.outgoingLinks.map(({ linkEntityTypeId, destinationEntityId }) => ({
-      webId,
-      linkData: {
-        leftEntityId: entity.metadata.recordId.entityId,
-        rightEntityId: destinationEntityId,
-      },
-      entityTypeIds: [linkEntityTypeId],
-      properties: { value: {} },
-      provenance,
-      draft: false,
-      relationships: [
-        {
-          relation: "administrator",
-          subject: {
-            kind: "account",
-            subjectId: params.authentication.actorId,
-          },
-        },
-        {
-          relation: "setting",
-          subject: { kind: "setting", subjectId: "administratorFromWeb" },
-        },
-        {
-          relation: "setting",
-          subject: { kind: "setting", subjectId: "updateFromWeb" },
-        },
-        {
-          relation: "setting",
-          subject: { kind: "setting", subjectId: "viewFromWeb" },
-        },
-      ],
-    })),
-  );
+  const linkEntities =
+    params.outgoingLinks.length > 0
+      ? await HashEntity.createMultiple(
+          graphApiClient,
+          { actorId: params.authentication.actorId },
+          params.outgoingLinks.map(
+            ({ linkEntityTypeId, destinationEntityId }) => ({
+              webId,
+              linkData: {
+                leftEntityId: entity.metadata.recordId.entityId,
+                rightEntityId: destinationEntityId,
+              },
+              entityTypeIds: [linkEntityTypeId],
+              properties: { value: {} },
+              provenance,
+              draft: false,
+              relationships: createLinearHashEntityAuthRelationships({
+                machineActorId: params.authentication.actorId,
+              }),
+            }),
+          ),
+        )
+      : [];
 
   // TODO: allow creating policies alongside entity creation
   //   see https://linear.app/hash/issue/H-4622/allow-creating-policies-alongside-entity-creation
-  for (const createdEntity of [entity, ...linkEntities]) {
+  for (const createdEntity of linkEntities) {
     const createdEntityUuid = extractEntityUuidFromEntityId(
       createdEntity.entityId,
     );
     await createPolicy(graphApiClient, params.authentication, {
```

Comment:
see https://github.com/hashintel/hash/pull/7405
```suggestion
    // TODO: allow creating policies alongside entity creation
    //   see https://linear.app/hash/issue/H-4622/allow-creating-policies-alongside-entity-creation
    await createPolicy(graphApiClient, params.authentication, {
```

Reply:
This one already appears two lines above 😁

Reply:
Fair enough 😅